src/content/rest/__tests__/restApi.test.ts

```diff
@@ -366,6 +366,43 @@ describe("rest api", () => {
         expect((await search(searchForProduct, opts)).total).toBe(1);
       });
 
+      it("results should contain description if path is provided in model", async () => {
+        const description = "description-test";
+
+        await create("news", {
+          slug: "foo-bar",
+          title: description
+        });
+
+        await create("articleNews", {
+          slug: "foo-bar",
+          title: description
+        });
+
+        expect(
+          (await search(description, {
+            published: false,
+            linkableOnly: false
+          })).total
+        ).toBe(2);
+
+        expect(
+          (await search(description, {
+            published: false,
+            linkableOnly: false,
+            includeModels: ["news"]
+          })).items[0]
+        ).toMatchObject({ description });
+
+        expect(
+          (await search(description, {
+            published: false,
+            linkableOnly: false,
+            includeModels: ["articleNews"]
+          })).items[0]
+        ).not.toMatchObject({ description });
```

Comment:
I would extract this second assertion in a separate test. "it should not contain description in result when no path is provided in model"
And also change the assertion to 
```ts
expect((await search/* ... */).items[0].description).toBe(undefined);
```